barretenberg/ts/src/types/fields.ts

```diff
@@ -15,7 +15,7 @@ export class Fr {
     const valueBigInt = typeof value === 'bigint' ? value : toBigIntBE(value);
 
     if (valueBigInt > Fr.MAX_VALUE) {
-      throw new Error(`Fr out of range: ${valueBigInt}`);
+      throw new Error(`Value 0x${valueBigInt.toString(16)} is greater or equal to field modulus.`);
```

Comment:
Unified the message here with `Fr` from foundation package since I am checking the messages in `tagged_log.ts`. Would probably be good to tackle [this one](https://github.com/AztecProtocol/aztec-packages/issues/4189)